src/minigames/minigame.ts

```diff
@@ -7,6 +7,14 @@ export interface MinigameDelegate {
 /** Base class for all minigames to extend. */
 export class Minigame {
   protected container: Container;
+  /**
+   * How long this minigame should go for before timing out
+   * and failing the player. In milliseconds. Set to
+   * undefined for a minigame with no time limit.
+   */
+  protected lifetime = 10_000;
+  protected elapsedMS = 0;
+  private ticker: Ticker;
```

Comment:
We just need to make sure this doesn't clash with the ticker defined in #18 , I think we can remove the ticker there and make it use this one?